isaric/parsers/isaric-rapid.json

```diff
@@ -387,54 +387,41 @@
       ]
     },
     "treatment_neuraminidase": {
-      "comment": "TODO: Create combination of combined types and conditionals here (depends on above attribute).",
       "description": "Antiviral type",
       "combinedType": "any",
       "fields": [
         {
-          "field": "antiviral_cmtrt",
+          "field": "antiviral_cmtrt___5",
```

Comment:
We should probably do this for CCPUK as well.

Reply:
This works where we only want to record info for a single antiviral agent (Neuraminidase in this case), setting the attribute to True/False - if we want to store information about all the different agents (and for corticosteroids & experimental treatments in rapid) we'll need a different approach. That said - I can implement this approach in the ccpuk parser for now

Reply:
That will be helpful.